src/chrome/content/rules/Mudah.my.xml

```diff
@@ -0,0 +1,70 @@
+<!--
+	Non-functional subdomains:
+
+		- mudah.my	(redirects to invalid "www.mudah.my/www.mudah.my/")
+		- 20www	(t)
+		- www.beta	(t)
+		- blog	(r)
+		- cari	(t)
+		- chat-preprod	(r)
+		- cimbpropertymart	(t)
+		- com	(t)
+		- contest	(t)
+		- www.contest	(t)
+		- edm	(r)
+		- fixedad	(t)
+		- googlewww	(t)
+		- hydra1-staging	(i)
+		- img	(t)
+		- img2	(t)
+		- www.img2	(t)
+		- mail	(t)
+		- malaysia	(t)
+		- mma	(t)
+		- motortakaful	(t)
+		- ooglewww	(t)
+		- panel	(m)
+		- stompinsabah	(t)
+		- survey	(t)
+		- ww	(t)
+		- www.ww	(t)
+		- ww43	(t)
+		- wwa	(t)
+		- www1	(t)
+		- www2	(t)
+		- www9	(t)
+		- wwww	(t)
+		- www.wwww	(t)
+		- wwwww	(t)
```

Comment:
Wildcard DNS (all `(t)`)

Reply:
A comment about the wildcard should be added.

Reply:
Done